src/content/docs/en/guides/astro-db.mdx

```diff
@@ -13,7 +13,7 @@ Astro DB is a fully-managed SQL database designed exclusively for Astro. Develop
 
 ## Installation
 
-Add Astro DB to a new or existing Astro project (requires `astro@4.5` or later) with the [`@astrojs/db` integration](/en/guides/integrations-guide/db/) (`v0.8.0` or later). Astro includes a built-in `astro add` command to automate this setup process for you.
+Add Astro DB to a new or existing Astro project (requires `astro@4.5` or later) with the [`@astrojs/db` integration](/en/guides/integrations-guide/db/) (`v0.8.1` or later). Astro includes a built-in `astro add` command to automate this setup process for you.
```

Comment:
Since v0.8.0 had a problem with exported types, which has been fixed in [v0.8.1](https://github.com/withastro/astro/releases/tag/%40astrojs%2Fdb%400.8.1), I think we should suggest that readers use v0.8.1 or later.